### ISSUE TITLE
"Digital Marketing Agency" theme of Elementor is causing delays

### ISSUE THREAD

**Issue by kiddingguy** [elementor/elementor#24792]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

After creating a website with the "Digital Marketing Agency" theme of Elementor on Google Search Console I get the following message:

<div class="elementor-element elementor-element-7019e30c e-flex e-con-boxed e-con e-pa…" data-id="7019e30c" data-element_type="container" data-settings="{"background_background":"gradient","shape_divider_bottom":"waves","conten…" data-e-bg-lazyload="" data-core-v316-plus="true"> 

![GoogleSearchConsole-github](https://github.com/elementor/elementor/assets/156768456/e7b741d2-e30d-4d6e-9afb-52f713ea7155)

### Steps to reproduce

1. Apply website with the "Digital Marketing Agency" theme
2. Adjust the header with a new image and text
3. Go to Google Search Console and do a speedtest of the website
4. The (error) message appears
5. Ranking of site in Google (on mobile) drops to 60% [not high] 

### Expected behavior

The major drop in ms should not be the case.
On desktop the numbers are okay.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.6.16
	PHP Version: 8.2.13
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.2
	Site URL: https://www.... .nl
	Home URL: https://www.... .nl
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /insights/%category%/%postname%/
	Language: en-US
	Timezone: Europe/Amsterdam
	Admin Email: ...@... .nl
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Twenty-Four
	Version: 1.0
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36

== Active Plugins ==
	All-in-One WP Migration
		Version: 7.79
		Author: ServMask

	Autoptimize
		Version: 3.1.10
		Author: Frank Goossens (futtta)

	Cookie Notice & Compliance for GDPR / CCPA
		Version: 2.4.13
		Author: Hu-manity.co

	Elementor
		Version: 3.18.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.18.2
		Author: Elementor.com

	Favicon by RealFaviconGenerator
		Version: 1.3.28
		Author: Philippe Bernard

	Folders Pro
		Version: 3.0.2
		Author: Premio

	Google Analytics for WordPress by MonsterInsights
		Version: 8.23.1
		Author: MonsterInsights

	Redirection
		Version: 5.4.1
		Author: John Godley

	Schema - All In One Schema Rich Snippets
		Version: 1.6.8
		Author: Brainstorm Force

	Site Offline
		Version: 1.5.6
		Author: Chandra Shekhar Sahu

	Social Warfare
		Version: 4.4.5.1
		Author: Warfare Plugins

	Title Remover
		Version: 1.2.1
		Author: WPGurus

	WP Super Cache
		Version: 1.11.0
		Author: Automattic

	XML Sitemap Generator for Google
		Version: 4.1.18
		Author: Auctollo

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Elements Usage ==
	
	archive : 2
		archive-posts : 2
		button : 2
		container : 10
		heading : 6
		image : 2
		theme-archive-title : 2
	footer : 5
		button : 2
		container : 16
		form : 1
		heading : 5
		icon-list : 2
		image : 2
		social-icons : 1
		spacer : 1
		text-editor : 9
	header : 3
		container : 6
		icon-list : 3
		nav-menu : 1
		social-icons : 1
		theme-site-logo : 3
	page : 1
		container : 29
		heading : 10
		icon-box : 12
		image : 3
		image-carousel : 1
		text-editor : 2
	popup : 2
		container : 2
		form : 1
		heading : 2
		icon : 1
		image : 1
	single : 2
		button : 1
		container : 7
		form : 1
		heading : 6
		image : 1
		post-info : 1
		post-navigation : 1
		posts : 1
		share-buttons : 1
		spacer : 2
		theme-post-content : 1
		theme-post-excerpt : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	wp-post : 14
		blockquote : 1
		heading : 14
		image : 20
		text-editor : 44
	wp-page : 4
		button : 4
		container : 27
		heading : 10
		icon-box : 12
		image : 3
		image-carousel : 1
		text-editor : 6
		video : 1
	single-page : 4
		container : 48
		divider : 6
		heading : 34
		image : 6
		media-carousel : 2
		menu-anchor : 8
		text-editor : 26
	error-404 : 2
		button : 2
		container : 6
		heading : 6
		image : 6


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active


== Elementor Experiments ==
	Optimized DOM Output: Inactive
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Inactive by default
	Default to New Theme Builder: Active
	Editor Top Bar: Inactive by default
	Optimized Gutenberg Loading: Active by default
	Build with AI: Inactive by default
	Landing Pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Optimize Image Loading: Inactive
	Global Style Guide: Active by default
	Page Transitions: Active
	Notes: Active
	Form Submissions: Active
	Scroll Snap: Inactive
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
Log: showing 20 of 312024-01-12 13:26:10 [info] elementor-pro::elementor_pro_updater Started 
2024-01-12 13:26:10 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-01-12 13:26:10 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-01-12 13:26:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.1',
  'to' => '3.18.2',
)]
2024-01-12 14:02:02 [info] Uploading Kit:  [array (
  'kit_id' => '631863428a66fa001de421f4',
  'referrer' => 'kit-library',
)]
2024-01-12 14:09:12 [info] Selected import runners: site-settings, plugins, templates, taxonomies, elementor-content, wp-content, elements-default-values 
2024-01-12 14:09:14 [info] Import runner completed: site-settings ✓ 
2024-01-12 14:09:14 [info] Import runner completed: plugins ✓ 
2024-01-12 14:09:29 [info] Import runner completed: templates ✓ 
2024-01-12 14:09:31 [info] Import runner completed: taxonomies ✓ 
2024-01-12 14:10:18 [info] Import runner completed: elementor-content ✓ 
2024-01-12 14:10:25 [info] Import runner completed: wp-content ✓ 
2024-01-13 14:34:38 [info] Uploading Kit:  [array (
  'kit_id' => '631863428a66fa001de421f4',
  'referrer' => 'kit-library',
)]
2024-01-13 14:35:08 [info] Selected import runners: site-settings, plugins, templates, taxonomies, elementor-content, wp-content, elements-default-values 
2024-01-13 14:35:08 [info] Import runner completed: site-settings ✓ 
2024-01-13 14:35:09 [info] Import runner completed: plugins ✓ 
2024-01-13 14:35:13 [info] Import runner completed: templates ✓ 
2024-01-13 14:35:14 [info] Import runner completed: taxonomies ✓ 
2024-01-13 14:35:16 [info] Import runner completed: elementor-content ✓ 
2024-01-13 14:35:16 [info] Import runner completed: wp-content ✓ 

PHP: showing 7 of 7PHP: 2023-12-09 03:13:58 [notice X 11][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/utils.php(727): class type strtolower()
#2: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/call-to-action/widgets/call-to-action.php(1594): Elementor\Utils :: validate_html_tag()
#3: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/base/controls-stack.php(2297): ElementorPro\Modules\CallToAction\Widgets\Call_To_Action -> render()
#4: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/base/widget-base.php(614): Elementor\Controls_Stack -> render_by_mode()
',
)]
PHP: 2023-12-09 03:18:51 [notice X 8][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/utils.php(727): class type strtolower()
#2: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/call-to-action/widgets/call-to-action.php(1593): Elementor\Utils :: validate_html_tag()
#3: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/base/controls-stack.php(2297): ElementorPro\Modules\CallToAction\Widgets\Call_To_Action -> render()
#4: /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/base/widget-base.php(614): Elementor\Controls_Stack -> render_by_mode()
',
)]
PHP: 2024-01-12 13:42:06 [notice X 2][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-12 14:15:20 [warning X 3][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::179] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-12 14:36:09 [notice X 9][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-13 11:47:50 [notice X 1][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/includes/template-library/sources/local.php::780] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-13 16:24:01 [error X 2][/var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::312] Uncaught Error: Call to a member function update_meta() on null in /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:312
Stack trace:
#0 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(185): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-content/plugins/elementor/core/common/modules/ajax/module.php(176): call_user_func()
#3 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-includes/class-wp-hook.php(324): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()
#5 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /var/www/vhosts/... .nl/webspace/httpdocs/... .nl-wordpress/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 4 of 4JS: 2024-01-12 14:25:59 [error X 7][https://www.... .nl/wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=6.4.2:2:28760] Cannot read properties of undefined (reading 'value') 
JS: 2024-01-13 14:17:44 [error X 4][https://www.... .nl/wp-content/plugins/elementor-pro/assets/js/frontend.min.js?ver=3.18.2:2:12705] Failed to execute 'observe' on 'IntersectionObserver': parameter 1 is not of type 'Element'. 
JS: 2024-01-13 14:40:44 [error X 4979][https://www.... .nl/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:14799] Cannot read properties of null (reading 'clone') 
JS: 2024-01-13 14:40:46 [error X 110][https://www.... .nl/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:19552] Cannot read properties of null (reading 'changestop') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by benravetta** [comment]
I can confirm this is not isolated to template issue: I am building a custom theme on Hello, and this is my current LCP: https://i.imgur.com/klljHaM.png
